scrapper.py

```diff
@@ -35,7 +50,25 @@ def find_articles(self):
         """
         Finds articles
         """
-        pass
+        headers = {
+            'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 '
+                          '(KHTML, like Gecko) Chrome/99.0.4844.51 Safari/537.36 Edg/99.0.1150.36',
+            'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,'
+                      'image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9',
+            'Accept-Encoding': 'gzip, deflate',
+            'Accept-Language': 'ru,en-GB;q=0.9,en;q=0.8,en-US;q=0.7'
+        }
+
+        for seed_url in self.seed_urls:
+            response = requests.get(url=seed_url, headers=headers)
+            if not response.ok:
+                print('request failed')
+
+            soup = BeautifulSoup(response.text, 'lxml')
+            main_block_bs = soup.find('div', {'class': 'two_thirds'})
```

Comment:
Leave extracting links from soup object to `_extract_url`, that what this method is for

Reply:
Done